src/compressed_tensors/quantization/lifecycle/initialize.py

```diff
@@ -55,16 +57,29 @@ class KVCacheScaleType(Enum):
     VALUE = "v_scale"
 
 
+ALL_QPARAM_KEYS = [KVCacheScaleType.KEY.value, KVCacheScaleType.VALUE.value] + [
```

Comment:
Ideally this list lives in some central location and we can just pull it in.
The way it is now, we can easily add in other params and have it missed here.

This is already stored in the compression_param_names (although the compressor is not yet set-up in the lifecyle).

Reply:
I really think that `compression_param_names` is bad practice left over from the pre-meta model days. I'd prefer to have this list listed here.

Reply:
This list should be in some central location as I mentioned above. Ideally easily used by the compressors /  init if and when required.

We need some standard source of truth

Reply:
updated based on slack conversation, moved to a higher-level module so it can be used more broadly. ready for another review